mscore/musescore.cpp

```diff
@@ -6006,7 +6006,10 @@ int main(int argc, char* av[])
             QString temp = parser.value("r");
             if (temp.isEmpty())
                    parser.showHelp(EXIT_FAILURE);
-            preferences.setTemporaryPreference(PREF_EXPORT_PNG_RESOLUTION, temp.toDouble());
+            bool ok = false;
+            double res = temp.toDouble(&ok);
+            if (ok)
+                  preferences.setTemporaryPreference(PREF_EXPORT_PNG_RESOLUTION, res);
```

Comment:
Well, since we are at it, let's add a non fatal (?) error message?

Reply:
Something simple as this?
<cpp>
            else
                  fprintf(stderr, "PNG resolution not recognized, using default settings from preferences instead\n");
</cpp>

Reply:
Yes.

Reply:
quoting the value that we parsed ?

Reply:
Yes.

Reply:
Will do the same for the 2.3 PR

Reply:
Also putting out the usage message, using `parser.helpText();`? Before or after the error message?

Reply:
maybe too much no ?